src/EventStore.Core/ClusterVNodeStartup.cs

```diff
@@ -279,7 +281,14 @@ private class StatusCheck {
 						.MapMiddlewareGet("live", inner => inner.Use(Live)));
 
 			private MidFunc Live => (context, next) => {
-				context.Response.StatusCode = _startup._ready ? 204 : 503;
+				if (_startup._ready) {
+					if (context.Request.Query.ContainsKey("liveCode"))
+						context.Response.StatusCode =
+							Int32.Parse(context.Request.Query.ToDictionary().GetValueOrDefault("liveCode")[0]!);
```

Comment:
This needs error handling for if the user sends something that doesn't parse to an int. You can use `int.TryParse` to do this in a cleaner way. Similarly, you can use `TryGetValue` above to get the expected code from the dictionary.
For example:

```
if (context.Request.Query.TryGetValue("liveCode", out var expected)
    && int.TryParse(expected, out var statusCode)) {
        context.Response.StatusCode = statusCode;
}
```

Reply:
Done